### DISCUSSION TITLE
Invoke-FalconAdminCommand wait time question?

### DISCUSSION THREAD

**Discussion by mcj323s** [CrowdStrike/psfalcon#403] [General]
Sometimes running Invoke-FalconAdminCommand with the -wait causes and indefinite wait.  Is there a way to limit the wait time to end the RTR session after 5 minutes or so? 
I have specific set of commands that use Invoke-FalconAdminCommand in order. I have tried Invoke-FalconRTR but the mkdir, cd, put, runscript (run the powershell install) demand their own session.

**Comment by bk-cs** [comment]
[Invoke-FalconDeploy](https://github.com/crowdstrike/psfalcon/wiki/Invoke-FalconDeploy) is designed to do all those things.

There is not a built-in break for the `-Wait` parameter. You can cancel the command while it's waiting and the session will expire once the defined timeout (30 seconds if not set otherwise) is reached.
